src/js/api/posts/create.test.js

```diff
@@ -1,14 +1,20 @@
 import { create } from './create.js';
-import { apiBaseFetch } from '../apiBaseFetch.js';
 
-jest.mock('../apiBaseFetch.js');
 
 describe('create', () => {
+  
+  global.fetch = jest.fn(() => {});
```

Comment:
Global fetch is being called inside the describe block, I think this leads to timing issues.
It should be moved to the top level to ensure its set up before the "create" function is imported and used.

Reply:
Good point, i've moved the mock outside of the describe block!